packages/endomoat/test/scenarios.spec.js

```diff
@@ -0,0 +1,65 @@
+import test from 'ava'
+// @ts-expect-error - needs types
+import { loadScenarios } from 'lavamoat-core/test/scenarios/index.js'
+// @ts-expect-error - needs types
+import { runAndTestScenario } from 'lavamoat-core/test/util.js'
+import { runScenario } from './helpers.js'
+
+const GLOBAL_WRITE_REGEX = /".+?":"write"/g
+
+// @ts-expect-error - needs types
+function policyHasWritableGlobal(scenario) {
+  return (
+    GLOBAL_WRITE_REGEX.test(JSON.stringify(scenario.config)) ||
+    GLOBAL_WRITE_REGEX.test(JSON.stringify(scenario.configOverride))
+  )
+}
+
+const SKIP_SCENARIOS = new Set([
+  'globalRef - check default containment',
+  'globalRef - Webpack code in the wild works',
+])
+
+test('Run scenarios', async (t) => {
```

Comment:
this runs all the scenarios from `core` which haven't been explicitly skipped due to something-not-yet-implemented or other reasons.  the stuff with `???` is skipped in `lavamoat-node` as well.